src/Symfony/Component/Serializer/Mapping/ClassMetadata.php

```diff
@@ -28,7 +28,7 @@ class ClassMetadata implements ClassMetadataInterface
     public $name;
 
     /**
-     * @var AttributeMetadataInterface[]
+     * @var array
```

Comment:
I see, so technically this is a bugfix, isn't it? I would say let's improve the PHPdoc in all versions where this applies then.